NEXT_CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## Release v0.267.0
 
 ### Notable Changes
```

Comment:
We don't have formal criteria for "notable changes" but usually we place breaking changes there or regression fixes. Adding resources goes into Bundles.

I think we should have "### Python" section for all Python changes, WDYT?

Reply:
It should live under the bundle section. Incremental features go there. Massive features, breaking changes, regression fixes, etc, go under "Notable changes".

Reply:
👍 , fixed